### PR TITLE
added entry for AxelSpringerSE

### PR DESCRIPTION
**Pull request by renebaudisch**

### CODE REVIEW

File: first_party_sets.JSON
Comment (ratchanonsuttawas) [inline first_party_sets.JSON:1]: db706e830127c271f8e41fdb78d33007cc7abb8a

**Review by ratchanonsuttawas — COMMENTED** [review]
Good

**Comment by renebaudisch** [comment]
is already done by
https://github.com/GoogleChrome/related-website-sets/pull/23

File: first_party_sets.JSON
Comment (ratchanonsuttawas) [reply first_party_sets.JSON:1]: > db706e830127c271f8e41fdb78d33007cc7abb8a

**Review by ratchanonsuttawas — COMMENTED** [review]
Review these changes db706e830127c271f8e41fdb78d33007cc7abb8a

File: first_party_sets.JSON
Comment (ratchanonsuttawas) [inline first_party_sets.JSON:1]: >Review these changes db706e830127c271f8e41fdb78d33007cc7abb8a

**Review by ratchanonsuttawas — COMMENTED** [review]
Review these changes

**Pull request closed without merging**